resources/views/email.blade.php

```diff
@@ -196,7 +197,7 @@
                 <footer>
                     <div class="footer-top">
                         <div class="footer-logo">
-                            <img src="{{ asset('identification/logo.svg') }}" alt="Blumilk logo"/>
+                            <img src="https://blumilk.pl/identification/logo.svg" alt="Blumilk logo"/>
```

Comment:
Should be revert to `asset` which automatically uses APP_URL.
Image has been blocked by Authelia.
I propose to set Authelia only for admin panel for beta too.